parsl/tests/configs/local_radical.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from parsl.config import Config
+from parsl.executors.radical import RadicalPilotExecutor
+from parsl.executors.radical import ResourceConfig
+
+# This is temporary; once everything is merged, we will use Parsl instead of
+# this fork.
+parsl_src = "pip install git+https://github.com/AymenFJA/parsl.git"
```

Comment:
I think this open issue (to do with cloud execution) probably applies here: https://github.com/Parsl/parsl/issues/1137

We moved away from "grid/cloud" style execution, but as isolated environments that aren't grid/cloud (eg this, and containers) become more prominent it's something that's interesting again.

Reply:
a rough thought I've had about this before:

parsl has a `get_version` which tries to figure out some coherent version by looking at itself: are you running from a git repo? what's your version string from parsl.version (as mutated by the parsl release script)?

Maybe that's interesting and not too hard to adapt into function which can give an "install me!" `pip` command - that would fit in this line of code but also make sense for general users (including outside of radical, for example in containers, for other parsl non-radical users?)

Do you think that would work in this case for radical? If so, I'll try to code something up in a separate PR.

Reply:
(eg you'd write:    `parsl_src = parsl.install_command_for_this_version()`)

Reply:
> (eg you'd write: `parsl_src = parsl.install_command_for_this_version()`)

Well that can be helpful for RP partially. This is also related to how Parsl CI is doing the tests. Parsl CI tests do not create a virtual env and performs the installation directly on the native Python version of the VM. If we have a virtual env created before starting the tests and pointing RP to it, it will lead to:

1. RP will be able to use that env and cut short the bootstrapping time to 90% less.
2. We will not need to specify `parsl_src = "pip install git+https://github.com/AymenFJA/parsl.git"` at all.
3. This will reduce the produced logs on the agent side dramatically and eventually will help to reduce the `artifacts` to way less size and collection time.